dice/apps/games/tests/test_views/test_room_views.py

```diff
@@ -88,7 +88,7 @@ def test_join_room_user_exists(self):
             f'/api/v1/rooms/{self.room.id}/join/',
             format='json',
         )
-        self.assertEqual(response.status_code, 403)
+        self.assertEqual(response.status_code, 409)
```

Comment:
Always try to avoid magic numbers, instead of status codes you can use one of the following:

+ [`rest_framework.status`](https://www.django-rest-framework.org/api-guide/status-codes/)
+ [`http.HttpStatus`](https://docs.python.org/3/library/http.html#http.HTTPStatus)

You can correct it in this PR